packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -191,6 +191,8 @@ function Await(props: AwaitComponentProps) {
         >
             {props.brandLogo && <img src={props.brandLogo} alt={props.type} className="adyen-checkout__await__brand-logo" />}
 
+            {props.showAmount && <div className="adyen-checkout__await__amount">{i18n.amount(props.amount.value, props.amount.currency)}</div>}
```

Comment:
Is there a use-case where the amount is zero? If so, do we still display it here?

I believe that we have similar logic for QR code and [we don't display it if amount is zero](https://github.com/Adyen/adyen-web/blob/main/packages/lib/src/components/internal/QRLoader/QRLoader.tsx#L210)

Reply:
I can add that logic, however in this case `showAmount` is configurable per component. So all the other Await based components are not going to display amount.